prime-angular-frontend/src/app/shared/components/forms/address-form/address-form.component.html

```diff
@@ -12,7 +12,7 @@
 </app-address-form-autocomplete>
 
 <div [formGroup]="form"
-     [hidden]="!showAddressFields">
+     [hidden]="!showAddressFields && form.untouched">
```

Comment:
> Feels like if it's possible to avoid adding this we should. I would have thought that if you wanted the form field to be displayed you would set `showAddressFields = true` so whether it is displayed or not is independent of form state.

When the form is hidden and user click "save & continue", the validations on the form wont show. Adding form.untouched is quick hack.

Reply:
I realize what it does, but the reason this looks odd to me is due to AddressForm being a base component.  By doing this each component throughout the application is affected with hopefully no side-effects, and in order to consume the component you need to know that the state can control visibility not your choice of bindings.